codec/src/main/java/io/netty/handler/codec/compression/JdkZlibDecoder.java

```diff
@@ -39,6 +39,7 @@
 
     // GZIP related
     private final ByteBufChecksum crc;
+    private boolean decompressConcatenated;
```

Comment:
final

Reply:
Making this `final` forces boilerplate. Because of the `switch` in the constructor, the compiler thinks that the variable is not initialized, and it has to be explicitly initialized to `false` in all cases. I don't see any benefit in doing so since it's already `private`. WDYT?

Reply:
I would prefer to make it final if it will never be changed. This may also allow optimisations for the JIT / JVM

Reply:
Done.